components/com_newsfeeds/tmpl/newsfeed/default.php

```diff
@@ -42,7 +43,7 @@
 		<?php if ($this->params->get('display_num')) : ?>
 		<h1 class="<?php echo $direction; ?>">
 			<?php echo $this->escape($this->params->get('page_heading')); ?>
-		</h1>
+ 		</h1>
```

Comment:
```suggestion
		</h1>
```

remove extra leading space